data-serving/scripts/prune-uploads/requirements.txt

```diff
@@ -0,0 +1 @@
+pymongo
```

Comment:
don't we want to use `poetry` everywhere now?

Reply:
(besides which, it'd be good to pin the version)

Reply:
I prefer requirements.txt for simple one liner scripts with few dependencies, can change to poetry if we want to maintain consistency. Advantage of requirements.txt is the setup code is one-liner unlike poetry which uses a `curl | python` style of installation.